hugo.yaml

```diff
@@ -148,10 +148,18 @@ params:
         url: 'https://groups.google.com/a/kubernetes.io/group/dev'
         icon: fa fa-envelope
         desc: Discussion and help from your fellow users
-      - name: Twitter
-        url: 'https://twitter.com/K8sContributors'
-        icon: fab fa-twitter
-        desc: Follow us on Twitter to get the latest news!
+      - name: LinkedIn
+        url: 'https://www.linkedin.com/company/kubernetes/'
+        icon: fab fa-linkedin
+        desc: Follow us on LinkedIn to get the latest news!
+      - name: Mastodon
+        url: 'https://hachyderm.io/@K8sContributors'
+        icon: fab fa-mastodon
+        desc: Follow us on Mastodon to get the latest news!
+      - name: Bluesky
+        url: 'https://bsky.app/profile/kubernetes.dev'
+        icon: fab fa-bluesky
+        desc: Follow us on Bluesky to get the latest news!
```

Comment:
Should we merge this before the Bluesky icon works? I'm unsure.

Reply:
One less thing to change later. I'm indifferent.

Reply:
If you drop it, I can definitely merge the change @chris-short. I feel like adding a broken link to Bluesky is technical debt (and potentially a source of issues, etc).

Reply:
Done.